### PR TITLE
Feature/update teams to projects

### PR DESCRIPTION
**Pull request by pswaine**

### CODE REVIEW

File: content/basics/terminology-and-concepts.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ Docker registries such as Docker Hub and Quay.io are used to store Docker images
 Kubernetes clusters are used to run Docker images. ContinuousPipe supports GCE, AWS or Kubernetes clusters on your own infrastructure.
 
 ## Concepts
-### Team
-A Team is used to group together users, code repository access tokens, Docker image registries and Kubernetes deployment clusters.
+### Projects
```
Comment (tonypiper) [inline content/basics/terminology-and-concepts.md]: `Project` ? (singular)

File: content/guides/laravel.md [hunk 2]
```diff
@@ -63,13 +63,13 @@ $ composer require predis/predis
...
 
 One last thing we need to do before we can begin using ContinuousPipe is configure which cluster we will use. You are free to configure as many clusters as you wish, in fact, its a good idea to have a separate cluster from production as you would for development and testing. 
 
 For now we have only got one cluster configured in ContinuousPipe. Lets assign that cluster name “main-cluster” to an environment variable which can be used with our build and deployment tasks.
 
-From within our Flow, click on configuration in the sidebar, create a new environment variable named `CLUSTER` with a value to match what you set as your cluster name/identifier. I have set my cluster name as `main-cluster`. Finally save the configuration.
+From within our flow, click on configuration in the sidebar, create a new environment variable named `CLUSTER` with a value to match what you set as your cluster name/identifier. I have set my cluster name as `main-cluster`. Finally save the configuration.
```
Comment (tonypiper) [inline content/guides/laravel.md]: the flow?

File: content/guides/symfony.md [hunk 3]
```diff
@@ -144,13 +144,13 @@ The ContinousPipe documentation has more information about [tasks]({{< relref "c
 
 ### Configuring the ContinuousPipe Console
 
-Before we can push any code to our repository, we need to ensure ContinuousPipe console is properly configured. Please refer to the ContinuousPipe Quick Start guide to setup your [team]({{< relref "quick-start/creating-a-team.md" >}}), [cluster]({{< relref "quick-start/configuring-a-cluster.md" >}}), [registry]({{< relref "quick-start/configuring-a-registry.md" >}}) and create your first [flow]({{< relref "quick-start/creating-a-flow.md" >}}).
+Before we can push any code to our repository, we need to ensure ContinuousPipe console is properly configured. Please refer to the ContinuousPipe Quick Start guide to setup your [project]({{< relref "quick-start/creating-a-project.md" >}}), [cluster]({{< relref "quick-start/configuring-a-cluster.md" >}}), [registry]({{< relref "quick-start/configuring-a-registry.md" >}}) and create your first [flow]({{< relref "quick-start/creating-a-flow.md" >}}).
```
Comment (tonypiper) [inline content/guides/symfony.md]: the ContinuousPipe console

**Comment by pswaine** [comment]
updates made @tonypiper

**Comment by sroze** [comment]
Wow, massive work 👏

**Pull request merged**